.github/workflows/synthetic-monitor.yml

```
@@ -64,6 +64,10 @@ jobs:
           NODE_PATH: backend/node_modules
         run: node scripts/synthetic-monitor.js
 
+      - name: Skip check (secret not configured)
+        if: ${{ secrets.SYNTHETIC_DONOR_SECRET_KEY == '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)